rosbag2_py/src/rosbag2_py/format_bag_metadata.cpp

```diff
@@ -157,10 +255,19 @@ std::string format_bag_meta_data(const rosbag2_storage::BagMetadata & metadata)
   info_stream << "Start:             " << format_time_point(start_time) <<
     std::endl;
   info_stream << "End:               " << format_time_point(end_time) << std::endl;
-  info_stream << "Messages:          " << metadata.message_count << std::endl;
+  info_stream << "Messages:          " << metadata.message_count - total_service_event_msg_count <<
+    std::endl;
   info_stream << "Topic information: ";
   format_topics_with_type(
     metadata.topics_with_message_count, info_stream, indentation_spaces);
 
+  if (!only_topic) {
+    info_stream << "Service:           " << service_info_list.size() << std::endl;
+    info_stream << "Service information: ";
+    if (service_info_list.size() != 0) {
```

Comment:
Comments from MichaelOrlov

```suggestion
    if (!service_info_list.empty()) {
```

clang-tidy recommends using the empty() method

Reply:
Yes. Updated.